benchmarks/query_middleware.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+var http = require('http')
+var Benchmark = require('benchmark')
+var suite = new Benchmark.Suite()
+
+var sr = {}
+
+var middleware = require('../lib/middleware/query')()
```

Comment:
The problem is to track how express performs. If the code changes (change the used libs), the contributor can track the performance changes too.
If you want, I'll remove this file.